python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -88,7 +90,12 @@ class ForcedPhotCoaddTask(ForcedPhotImageTask):
     dataPrefix = "deepCoadd_"
 
     def getExposure(self, dataRef):
-        name = self.config.coaddName + "Coadd_calexp"
+
+        if self.config.fakesAdded:
+            name = "fakes_" + self.config.coaddName + "Coadd_calexp"
```

Comment:
In general the python community seems to be going toward using the format method or f strings instead of the addition operator, so it might be useful to get in the habit. This code would then be
```
name = f"fakes_{self.config.coaddName}Coadd_calex"
```
What you have will run fine, just using the opportunity to maybe pass along some tips. There is a runtime performance you gain by using f strings or format vs concatenation, but in this case the micro optimization does not really matter

Reply:
Is it preferable to use this way or to have a consistent formatting throughout the file?

Reply:
https://developer.lsst.io/coding/intro.html?highlight=consistency#deviating-from-the-dm-style-guides (second bullet):

> Coding consistency is very important but sometimes the style guide doesn’t apply either due to lack of a definitive rule or the circumstances of the specific code segment logically dictate otherwise. When in doubt, use your best judgment or ask the lead developer.
> 
> Here are two plausible reasons to break a particular rule:
> - When applying the rule would make the code less readable, even for someone who is used to reading code that follows the rules.
> - To be consistent with surrounding code that also breaks it (maybe for historic reasons)—although this is also an opportunity to clean up someone else’s mess.